sdk/python/kfp/_credentials.py

```diff
@@ -12,16 +12,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import os
 import abc
 
 from kubernetes.client.configuration import Configuration
 
 
 __all__ = [
     "TokenCredentialsBase",
+    "ML_PIPELINE_SA_TOKEN_ENV",
+    "ML_PIPELINE_SA_TOKEN_PATH",
+    "ServiceAccountTokenVolumeCredentials",
 ]
 
 
+ML_PIPELINE_SA_TOKEN_ENV = "ML_PIPELINE_SA_TOKEN_PATH"
+ML_PIPELINE_SA_TOKEN_PATH = "/var/run/secrets/ml-pipeline/token"
```

Comment:
nit: avoid `ml-pipeline`

/var/run/secrets/kubeflow/pipelines/token WDUT? or sth you prefer

Reply:
Changed the value as you suggested